packages/components/src/components/autocomplete/autocomplete.jsx

```diff
@@ -299,7 +300,7 @@ const Autocomplete = React.memo(props => {
                         // marginTop: form.errors[field.name] ? 'calc(4px - 18px)' : '4px', // 4px is the standard margin. In case of error, the list should overlap the error
                     }),
                 }}
-                is_visible={should_show_list}
+                is_visible={should_show_list || is_list_visible}
```

Comment:
Can we give `should_show_list` an initial value of `is_list_visible` instead?

Reply:
we want the list to be visible always but `should_show_list` shows/hides the list on click of the input field in the component

Reply:
But `should_show_list` is a state, so what happens when the initial value of the state is true? Does it still wait for the user to click on the component?

Reply:
When the initial value is true, it will show the list initially when rendered but if the input field is clicked/focused `should_show_list` will hide the list which we don't want to happen.

Reply:
The `should_show_list` state is always initialised as false and changes on click of the input field. CMIIW

Reply:
yes correct @nijil-binary